6.bulk_Morphology_Elastic_Network/3.model_coefficients/scripts/1.model_coefficients.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import joblib
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
It looked like numpy and seaborn might not have been used. Consider removing them as imports unless needed for other work.

Reply:
Removed packages